lib/Onnxifi/Base.h

```diff
@@ -62,8 +64,13 @@ class Backend {
   /// \returns Execution Engine associated with the Backend.
   glow::ExecutionEngine &getEE() { return backendIdPtr_->getEE(); }
 
+  /// Run async using backend thread pool.
+  void runAsync(const std::function<void(void)> &fn);
+
 private:
   BackendIdPtr backendIdPtr_;
+  // ThreadPool instance for the backend.
+  ThreadPool threadPool_;
```

Comment:
we'd need to pass #workers in a ctor of backend and initialize threadPool_ properly.